Assets/QA/Tests/KeyboardMouseBasic/Scripts/ButtonStateHEX.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+using UnityEngine.UI;
+using ISX;
+
+
+[RequireComponent (typeof(Text))]
+public class ButtonStateHEX : MonoBehaviour {
```

Comment:
Please run Unity formatting tools on all C# code before committing. On OSX, run format.sh from the repo root. On Windows, run format.ps1 from there.